### PR TITLE
Add short version tag

### PR DESCRIPTION
**Pull request by tobiasge**
Adds a short version tag in the form of v$MAJOR.$MINOR. This tags
creates the possibillity to follow patch releases automatically with the
scheduled import option in Openshift and Kubernetes. The tag is only added for releases.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -34,6 +34,10 @@ if [ "${1}x" == "x" ] || [ "${1}" == "--help" ] || [ "${1}" == "-h" ]; then
   echo "  DOCKER_TAG The name of the tag which is applied to the image."
   echo "           Useful for pushing into another registry than hub.docker.com."
   echo "           Default: <DOCKER_ORG>/<DOCKER_REPO>:<BRANCH>"
+  echo "  DOCKER_SHORT_TAG The name of the short tag which is applied to the image."
+  echo "           Useful for pushing into another registry than hub.docker.com."
```
Comment (cimnine) [inline build.sh]: This line seems to be an accidental leftover.

Comment (tobiasge) [reply build.sh]: I have removed the line

File: build.sh [hunk 2]
```diff
@@ -165,4 +168,16 @@ if [ "${2}" == "--push" ] || [ "${2}" == "--push-only" ] ; then
   echo "⏫ Pushing '${DOCKER_TAG}"
   $DOCKER_CMD push "${DOCKER_TAG}"
   echo "✅ Finished pushing the Docker image '${DOCKER_TAG}'."
+  if [ "${TAG:0:1}" == "v" ]; then
```
Comment (cimnine) [inline build.sh]: I suggest to use bash's built-in regex here:

```bash
# TAG=v2.5.5

if [[ "${TAG}" =~ v([0-9]+)\.([0-9]+)\.[0-9]+ ]]; then
  MAJOR=${BASH_REMATCH[1]}
  MINOR=${BASH_REMATCH[2]}
  DOCKER_SHORT_TAG="${DOCKER_SHORT_TAG-${DOCKER_ORG}/${DOCKER_REPO}:${MAJOR}.${MINOR}}"
  # ...
```

This reduced the need for `cut` to be available and behaving in a certain manner.

Comment (tobiasge) [reply build.sh]: That is a good idea. I will make the change.

**Comment by tobiasge** [comment]
> It raises the question whether we should also have a `netboxcommunity/netbox-docker:2`. I have not come to a conclusion on this, but I'm sure you have thought about that as well. So I'm curious about why you decided not to build it.
I thought about that too. I decided against it, because of the way we use Netbox in our environment. We mainly use the API and because with the changes in minor version number there are always API changes, we can't automatically update across those.

**Comment by cimnine** [comment]
Thank you for pushing these changes. I think this is ready to merge. I would like to merge this when I have some time to spare as I don't know how this will effect the build on hub.docker.com. (I don't believe there will be any problem, but I'd rather have some time to look at it right away just in case.)

**Pull request merged**